src/core/github.js

```diff
@@ -42,6 +50,7 @@ export async function run(conf) {
       "Config option `[github](https://github.com/w3c/respec/wiki/github)` " +
       "is missing property `repoURL`.";
     pub("error", msg);
+    githubRejector(new Error(msg));
```

Comment:
Bit weird that a function name is a noun 👀

And could we merge `githubRejector` and `pub("error")` here?

Reply:
> Bit weird that a function name is a noun :eyes:

`rejectGithubPromise`?

> And could we merge githubRejector and pub("error") here?

Good idea!